docs/faq.md

```diff
@@ -148,11 +149,9 @@ See [Extended Usage](../jib-maven-plugin#extended-usage) for the `<container><fo
 
 See [Extended Usage](../jib-gradle-plugin#extended-usage) for the `container.format` configuration.
 
-### Can I define a custom entrypoint?
+### Can I define a custom entrypoint at runtime?
 
-The plugin attaches a default entrypoint that will run your application automatically.
-
-When running the image, you can override this default entrypoint with your own custom command.
+Normally, the plugin sets a default entrypoint for java applications, or lets you configure a custom entrypoint using the `container.entrypoint` configuration parameter. You can also override the default/configured entrypoint by defining a custom entrypoint when running the container.
```

Comment:
I feel like the last statement should be in the second paragraph? Or just have everything in a single paragraph.